sync_check_terraform/modules/sync_check/service/init.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+## Install dependencies
+dnf install -y dnf-plugins-core
+dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
+dnf install -y docker-ce docker-ce-cli containerd.io
+dnf install -y docker-buildx-plugin docker-compose-plugin docker-compose
+dnf install -y ruby
+dnf install -y ruby-devel
+dnf install -y gcc make
+dnf clean all
+gem install slack-ruby-client
+gem install sys-filesystem
```

Comment:
you can add multiple packages to install in one command, e.g., `dnf install ruby ruby devel gcc make ...`

Reply:
same as in `gem`